tests/spec/core/xref-spec.js

```diff
@@ -237,14 +237,14 @@ describe("Core — xref", () => {
   it("shows error if cannot resolve by data-cite", async () => {
     const body = `
       <section data-cite="svg">
-        <p><a id="link">symbol</a> twice in svg spec.</p>
+        <p id="test">[^symbol^] twice in svg spec.</p>
```

Comment:
We earlier treated elements as concepts. Now, element's have their own category (element types). So, updated test.